src/Sulu/Component/DocumentManager/Event/QueryCreateEvent.php

```diff
@@ -39,7 +40,7 @@ class QueryCreateEvent extends AbstractEvent
     private $primarySelector;
 
     /**
-     * @param string $innerQuery
+     * @param string|QueryInterface $innerQuery
```

Comment:
when does this happen? Can it instead be casted? Or do we also need to change the `getInnerQuery`  return type

Reply:
We could do that. I don't know the implications.

Reply:
lets keep it and change the `getInnerQuery` to `@return string|QueryInterface`

Reply:
Updated it